Check/DoctrineMongoDb.php

```diff
@@ -20,7 +20,7 @@ public function __construct(ManagerRegistry $registry, $connectionName = null)
     public function check()
     {
         $connection = $this->manager->getConnection();
-        $connection->connect();
+        $connection->isConnected();
```

Comment:
I assume so far relied on the `connect()` throwing an exception .. when was the `isConnected()` method added?

Reply:
No idea when it was added, but connect() doesn't throw an exception (anymore).

Reply:
ok so isConnected() was added 5 years ago .. but it returns a boolean .. so we need to make a conditional check if to return a Success() or a Failure() instance